### DISCUSSION TITLE
🚀 [REQUEST] - Does anyone use Stripe?

### DISCUSSION THREAD

**Discussion by onursonmez** [Oksydan/falcon#397] [General]
### Summary

Stripe form does not work on my own site. I have to use the redirect feature. I wonder if anyone uses Falcon theme and uses the stripe payment infrastructure? Can you write your site name if you have one? Maybe I can detect the error.

### Feature target required module/theme

theme

### Drawbacks

Stripe payment system not working

**Comment by Oksydan** [comment]
Hi @onursonmez,

Unfortunately, I do not use stipe. I cannot help with this problem. TBH, if the module was written correctly, there should be no problem. Probably the module creator did something weird like `.parent().parent().parent()` etc in the javascript code. In the Falcon theme we have much less html code and the structure is much simpler. 
I am not able to support every module in the world. 
I am turning this issue into a discussion.

**Comment by mohaidoss** [comment]
@onursonmez hi did u find a workaround ?

**Reply by mohaidoss** [reply]
Workaround is mentioned here. https://github.com/Oksydan/falcon/issues/391#issuecomment-2006849918
